### PR TITLE
[RR] Made some minor changes in README

### PR DESCRIPTION
**Pull request by Ananyaiitbhilai**
-Spelling of applications was corrected
-made changes in documentation instructions

### CODE REVIEW

**Comment by Ananyaiitbhilai** [comment]
@yatendra1999

**Comment by yatendra1999** [comment]
Don't make new PRs for the same topic.

File: README.md [hunk 1]
```diff
@@ -22,9 +22,24 @@ Memory Imaging for Linux Family by OpenLake scans through the virtual memory of
...
+3. Commit the changes of Doumentation folder in the branch by running the following commands in a sequence
+``` 
+ git add .
+```
+``` 
+ git commit -m "<your message>"
+```
+``` 
+ git push
+```
```
Comment (yatendra1999) [inline README.md]: Put all three into a single code block like 
```
c1
c2
```

**Review by yatendra1999 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Ananyaiitbhilai) [reply README.md]: Ok done.

**Comment by Ananyaiitbhilai** [comment]
Pls review now...I made the asked changes

**Comment by Ananyaiitbhilai** [comment]
> Don't make new PRs for the same topic.

Okk...sorry

**Comment by Ananyaiitbhilai** [comment]
@yatendra1999

**Review by yatendra1999 — APPROVED** [review]

**Pull request merged**